scripts/lis_hyperv_platform/lis-rpm-hyperv-tests-lisav2.ps1

```diff
@@ -0,0 +1,96 @@
+[CmdletBinding()]
+param(
+    [parameter(Mandatory = $true)]
+    [String] $JobId,
+    [parameter(Mandatory = $true)]
+    [String] $DistroVersion,
+    [parameter(Mandatory = $true)]
+    [String] $TestCategory,
+    [String] $TestArea,
+    [String] $TestNames,
+    [string] $TestTag,
+    [string] $TestPriority,
+    [String] $LISAImagesShareUrl,
+    [String] $LisUrl,
+    [String] $LisOldUrl,
+    [String] $ExcludeTests,
+    [String] $IncludeTests,
+    [string] $ExecutionTag,
+    [String] $Delay
+)
+
+foreach ($key in $MyInvocation.BoundParameters.keys) {
+    $value = (get-variable $key).Value
+    write-host "$key -> $value"
+}
+
+function Main {
+    if (!$TestCategory) { $TestCategory = "All" }
+    if (!$TestArea)     { $TestArea = "All" }
+    if (!$TestNames)    { $TestNames = "All" }
+    if (!$TestTag)      { $TestTag = "All" }
+    if (!$TestPriority) { $TestPriority = "All" }
```

Comment:
The line number 28 ~ 32 leads to run all if those params are not defined. It will cause the potential capacity issue with a few feature tests. Please refresh your design.

Reply:
Removed the code.